spec/features/catalog_spec.rb

```diff
@@ -199,6 +199,83 @@
   end
 end
 
+feature "Search and browse facet links for scientific name" do
```

Comment:
this feature name is slightly misleading, as it's more focused on browsing facet values, and not about discovering them via search. It also covers more than scientific name, and appears to include all the custom faceted subjects.

I suppose one option is to include a test inside this feature for confirming the facets display on a search (in this case for "QE8iWjhafTRpc My Object" and then rename to something more generic. Alternatively, we could call this feature something like "Browse facet links for custom subjects" and then have a separate test for asserting the search discovery? Whatever you think makes the most sense.